<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QG_ImageOptionsDialog</class>
 <widget class="QDialog" name="QG_ImageOptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Export Options</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="bgSize">
     <property name="title">
      <string>Bitmap Size</string>
     </property>
     <layout class="QGridLayout">
      <item row="2" column="1">
       <widget class="QLabel" name="lResolution">
        <property name="text">
         <string>Resolution:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QComboBox" name="cbResolution">
        <property name="editable">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>auto</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">15</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">20</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">25</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">50</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">75</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">100</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">150</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">300</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">1200</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="7" column="0" colspan="3">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="lWidth">
        <property name="text">
         <string>Width:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="lHeight">
        <property name="text">
         <string>Height:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLineEdit" name="leHeight">
        <property name="text">
         <string notr="true">480</string>
        </property>
        <property name="placeholderText">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLineEdit" name="leWidth">
        <property name="text">
         <string notr="true">640</string>
        </property>
        <property name="placeholderText">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="bgBackground">
     <property name="title">
      <string>Background</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QRadioButton" name="rbWhite">
        <property name="text">
         <string>White</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbBlack">
        <property name="text">
         <string>B&amp;lack</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="hspacer1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <spacer name="vspacer3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>51</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="bgColouring">
     <property name="title">
      <string>Colouring</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="rbBlackWhite">
        <property name="text">
         <string>Black / White</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbColoured">
        <property name="text">
         <string>Coloured</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="hspacer2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <spacer name="vspacer4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>45</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="bgBorders">
     <property name="title">
      <string>Borders</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="1" column="1">
       <widget class="QLineEdit" name="leLeftRight">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="leTopBottom"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lTopBottom">
        <property name="text">
         <string>Top / Bottom - Border:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lLeftRight">
        <property name="text">
         <string>Left / Right - Border:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <spacer name="vspacer5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="cbSameBorders">
        <property name="text">
         <string>set same size</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>leWidth</tabstop>
  <tabstop>leHeight</tabstop>
  <tabstop>rbWhite</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QG_ImageOptionsDialog</receiver>
   <slot>ok()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>528</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QG_ImageOptionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>528</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leWidth</sender>
   <signal>textChanged(QString)</signal>
   <receiver>QG_ImageOptionsDialog</receiver>
   <slot>sizeChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leHeight</sender>
   <signal>textChanged(QString)</signal>
   <receiver>QG_ImageOptionsDialog</receiver>
   <slot>sizeChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbResolution</sender>
   <signal>editTextChanged(QString)</signal>
   <receiver>QG_ImageOptionsDialog</receiver>
   <slot>resolutionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbSameBorders</sender>
   <signal>stateChanged(int)</signal>
   <receiver>QG_ImageOptionsDialog</receiver>
   <slot>sameBordersChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leLeftRight</sender>
   <signal>textChanged(QString)</signal>
   <receiver>QG_ImageOptionsDialog</receiver>
   <slot>borderChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
